### PR TITLE
✨ JS-608: Select by Tag Name

### PR DESCRIPTION
**Pull request by ct-aa**
# Related Tickets
- [JS-608](https://codeandtheory.atlassian.net/browse/JS-608)

# Description
Added the **Select by Tag Name** challenge solutions.

# Validation
* Open the **Select by Tag Name** folder.
* Open the app.js.
* The variable listItems should store all the li elements.

# Risks
N/A

### CODE REVIEW

**Review by lycoangelo — CHANGES_REQUESTED** [review]
Hi @ct-aa, please fix the error. There's no `li` tag.

**Comment by ct-aa** [comment]
> Hi @ct-aa, please fix the error. There's no `li` tag.

Thanks, @lycoangelo, This is now updated.

File: Select by Tag Name/app.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Complete the challenge by writing JavaScript below
+const colors = ["#C2272D", "#F8931F", "#FFFF01", "#009245", "#0193D9", "#0C04ED", "#612F90"];
+let listItems = document.getElementsByTagName('li');
```
Comment (lycoangelo) [inline Select by Tag Name/app.js]: Please use `const` since the value was not reassigned.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

File: Select by Tag Name/app.js (hunk 1 above)
Comment (ct-aa) [reply Select by Tag Name/app.js]: Thanks, @lycoangelo, This is now updated.

**Review by lycoangelo — APPROVED** [review]

**Pull request merged**